### ISSUE TITLE
ANNOUNCING: VSIX for VS2017 is now in the Gallery

### ISSUE THREAD

**Issue by JerryNixon** [Windows-XAML/Template10#1468]
At last. At long last.

https://marketplace.visualstudio.com/items?itemName=jerry-nixon.Template10TemplatePack

**Comment by syaifulnizamyahya** [comment]
Nice. Thanks.

**Comment by bwarring24** [comment]
UPDATE: I found out from #1442 that you need Windows SDK 10.0.14393.0 installed. You can get this from the Visual Studio 2017 Installer application.

I'm running Windows 10 Insider build 16199 and VS 2017 and whenever I make a new Template 10 Hamburger project I get a "project creation fail".

**Comment by intoxopox** [comment]
Same here. Missing project subtype.

**Comment by VagueGit** [comment]
I added the Template 10 Template Pack for VS2017 from the Visual Studio Marketplace. Error creating Minimal Template from Add New Project/C#/Template 10 Minimal
![image](https://cloud.githubusercontent.com/assets/17174099/26524697/3801bf88-437f-11e7-8248-646019baf350.png)
I have these
![image](https://cloud.githubusercontent.com/assets/17174099/26524706/6be02d4e-437f-11e7-8c41-7dc9566d3eb9.png)
I followed the link in the error to the [SDK Archive](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive) and 10.0.14393.0 does not appear to be available.

I have
Windows Version 10.0.15063
Visual Studio Professional Version 15.2 (26430.6) Release.
Template 10 Template Pack 1.19

If I create a new Project/Windows Universal/Blank App then add Template 10 from Nuget I get Template 10 Latest prerelease 1.1.13-preview-20161207. It installs OK.

How do I get to use Template 10 Template Pack 1.19?

**UPDATE** Starting a new project with a Blank App (Universal Windows) VS prompted to install the appropriate Windows SDK. After that I was able to create a new project by selecting C#/Template 10/Minimal.

So it looks like I am on my way!!
